MiniSpotify/MiniSpotify/Source/APIRequestor.cs

```diff
@@ -242,13 +261,17 @@ public string GetCurrentSongArtwork()
 
             if(m_spotifyWebAPI != null && m_spotifyWebAPI.GetPlayback().IsPlaying)
             {
-                if(m_spotifyWebAPI.GetPlayingTrack().HasError())
+                if(!m_spotifyWebAPI.GetPlayingTrack().HasError())
                 {
-                    //Error something
-                }
-                else
-                {
-                    imageURL = m_spotifyWebAPI.GetPlayingTrack().Item.Album.Images[0].Url;
+                    try
+                    {
+                        imageURL = m_spotifyWebAPI.GetPlayingTrack().Item.Album.Images[0].Url;
+                        t_albumimage = imageURL; //Set it as 'backup'
+                    }catch(Exception e)
+                    {
+                        imageURL = t_albumimage; //Internet failure and get last time album as backup
+                        Console.WriteLine(e.StackTrace);
+                    }
```

Comment:
Two points here;
* I'm not a fan of us assuming it's an internet failure in the comment, but that's fine for now - It's not important why it fails most of the time.

* We don't do anything about if there is an error, we're now just ignoring it completely. Let's make this atleast do a Console output for now.

Reply:
Comment is replaced and better stable explanation and removed the 'backup'